packages/service/lib/stackable.js

```diff
@@ -54,6 +55,24 @@ class ServiceStackable {
     await this.app.close()
   }
 
+  async build () {
+    // We don't use this.app.log as the app has not being initialized.
+    const logger = globalThis.platformatic.logger
```

Comment:
We should be handling the case this is not present

Reply:
This is not possible: the logger is instantiate in `worker/main.js`, which is the entrypoint for the worker thread.